### PR TITLE
Update goals.txt

### PR DESCRIPTION
**Pull request by ciaome2**

### CODE REVIEW

**Comment by twang** [comment]
Note to self: rubrics at https://github.com/twangcs/student-goals-ciaome2/pull/2 and https://github.com/twangcs/student-goals-ciaome2/pull/3

File: goals.txt [hunk 1]
```diff
@@ -16,6 +16,6 @@ Directions:
 
 BEGIN WRITING BELOW
 
-Goal 1:
-Goal 2:
-Partner:
+Goal 1: Getting to know about what computer science is and how it is used to solve life problems or how it is used in techonlogy.
+Goal 2: Getting good grades, but understanding fully. Also, imply to what I am doing in robotisc club so that I can under stand Software function of our robotics team.
```
Comment (twang) [inline goals.txt]: I think this rubric and goal looks great if we just remove the grade goal (thinking about causes we have control over)

**Comment by Gabrielxddd** [comment]
good goals i agree

File: goals.txt (hunk 1 above)
Comment (ciaome2) [reply goals.txt]: Thank you and I'll change my grade goal for rubric 2 and 3!

**Pull request merged**